docs/pages/teamshares/contributing.md

```diff
@@ -63,28 +63,28 @@ yarn link
 npm run build
 ```
 
-#### Within `shared-ui`:
+#### Within `design-system`:
 
 ```bash
+yarn link
 yarn link @teamshares/shoelace
 yarn install --force
 ```
 
 #### Within your Teamshares app:
 
 ```bash
-yarn link @teamshares/ui
-yarn link @teamshares/shoelace
+yarn link @teamshares/design-system
 yarn install --force
 ```
 
 You can string all the above commands together into a single abomination like so:
 
 ```bash
-(cd ../shoelace && npm run build) && (cd ../teamshares-ui && yarn link @teamshares/shoelace && yarn install --force) && yarn link @teamshares/ui && yarn link @teamshares/shoelace && yarn install --force
+(cd ../shoelace && npm run build) && (cd ../design-system && yarn link && yarn link @teamshares/shoelace && yarn install --force) && yarn link @teamshares/design-system && yarn install --force
 ```
 
-Once all that is set up, you'll start your app like normal, i.e. `rails server` and `yarn build:all`. After each change in Shoelace, you'll need to run `npm run build` to generate the production build that `shared-ui` picks up. If you're using `yarn build --watch` and `yarn build:css --watch`, you'll need to restart those after each change as well.
+Once all that is set up, you'll start your app like normal, i.e. `rails server` and `yarn build:all`. After each change in Shoelace, you'll need to run `npm run build` to generate the production build that `design-system` picks up. If you're using `yarn build --watch` and `yarn build:css --watch`, you'll need to restart those after each change as well.
```

Comment:
The canonical command for starting our apps is now `yarn dev`, and the watch commands below are no longer necessary. You also don't actually need to re-build on every change; that was a relic of the older version of Shoelace. We can replace this whole paragraph with:
```
Once all that is set up, you'll start your app like normal via `yarn dev`. Changes made in Shoelace should automatically be reflected in your app.
```

Reply:
okay! updated in the new rebased branch!